src/commands/QR.ts

```diff
@@ -39,6 +39,32 @@ export default class QR extends Command {
     );
   }
 
+  /**
+   * Generates and returns a QR code given the data, title, and subtitle.
+   *
+   * @param data The content to put in the QR code.
+   * @param title event name
+   * @param subtitle event description
+   * @returns newly generated QR code
+   */
+  public static generateQR(data: string, title: string, subtitle: string): QRCode {
+    return new QRCode({
+      text: data,
+      colorDark: '#000000',
+      colorLight: 'rgba(0,0,0,0)',
+      correctLevel: QRCode.CorrectLevel.H,
+      logo: 'src/assets/acm-qr-logo.png',
+      logoBackgroundTransparent: false,
+      backgroundImage: 'src/assets/background.png',
+      quietZone: 40,
+      title: title.substring(0, 36) === title ? title : title.substring(0, 36).concat('...'),
+      titleTop: -20,
+      titleBackgroundColor: 'transparent',
+      subTitle: subtitle,
+      subTitleTop: -5,
+    });
+  }
+
```

Comment:
Since all usages of this need toDataURL, can we just return the dataURL here in this function so we don't need to re-call it everywhere else?